src/Rendering/babylon.occlusionBoundingBoxRenderer.ts

```diff
@@ -0,0 +1,82 @@
+module BABYLON {
+    
+    export class OcclusionBoundingBoxRenderer {

```

Comment:
I would prefer adding paramters to actual boundingbox renderer instead of creating a new class